roles/tftp-server/tasks/main.yml

```diff
@@ -11,10 +18,27 @@
     remote_src: true
     creates: /srv/tftp/pxelinux.0
 
+- name: create dir for configs
+  file:
+    path: "{{ web_root }}/{{ item }}"
+    state: directory
+    recurse: true
+    # owner:  "{{ user_name }}"
+    # group:  "{{ user_name }}"
+  with_items:
+  - lc
+  - d-i/xenial
```

Comment:
There is now also d-i/jessie 
It made it nice for testing both flavors.  

currently the playbooks only setup one, so there will be an extra dir that might confuse someone.
Once I get these reworks merged, then I am going to build up my R&D setup that will use it. 
I'll try to maintain some separation of stable vs testing so that my testing can be easily avoided.